Bottom: 1a52029ef3f9280ed743643e7b6d86757872a206
Top:    3e39516c458a48fa0baf2a0cd48f011dc24ea120
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-12-07 19:40:12 +0000

Win32: Disable the SpaceNavigator code


---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index ba73c39..c65fa58 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -551,8 +551,8 @@ ghid_port_window_enter_cb (GtkWidget * widget,
       /* Make sure drawing area has keyboard focus when we are in it.
        */
       gtk_widget_grab_focus (out->drawing_area);
-      if (ghidgui->snavi != NULL)
-        snavi_set_led (ghidgui->snavi, TRUE);
+      //if (ghidgui->snavi != NULL)
+      //  snavi_set_led (ghidgui->snavi, TRUE);
     }
   ghidgui->in_popup = FALSE;
 
@@ -585,8 +585,8 @@ ghid_port_window_leave_cb (GtkWidget * widget,
       return FALSE;
     }
 
-  if (out->has_entered && !ghidgui->in_popup && ghidgui->snavi != NULL)
-    snavi_set_led (ghidgui->snavi, FALSE);
+  //if (out->has_entered && !ghidgui->in_popup && ghidgui->snavi != NULL)
+  //  snavi_set_led (ghidgui->snavi, FALSE);
 
   out->has_entered = FALSE;
 
diff --git a/src/hid/gtk/snavi.c b/src/hid/gtk/snavi.c
index 2f58f02..d5c5850 100644
--- a/src/hid/gtk/snavi.c
+++ b/src/hid/gtk/snavi.c
@@ -12,6 +12,7 @@
  *
  */
 
+#if 0
 #include <stdio.h>
 #include <errno.h>
 #include <stdlib.h>
@@ -21,9 +22,11 @@
 
 #include <linux/types.h>
 #include <linux/input.h>
+#endif
 
 #include <glib.h>
 
+#if 0
 #define test_bit(bit, array)  (array [bit / 8] & (1 << (bit % 8)))
 
 static void (*update_pan_cb)(int, int, int, gpointer);
@@ -150,6 +153,7 @@ gboolean snavi_event (GIOChannel   *source,
 
   return TRUE;
 }
+#endif
 
 
 GIOChannel *
@@ -159,6 +163,9 @@ setup_snavi (void (*update_pan)(int, int, int, gpointer),
              void (*button)(int, int, gpointer),
              gpointer data)
 {
+  return NULL;
+
+#if 0
   GIOChannel *snavi;
 
   update_pan_cb = update_pan;
@@ -186,4 +193,5 @@ setup_snavi (void (*update_pan)(int, int, int, gpointer),
     }
 
   return snavi;
+#endif
 }
